relay-event-normalization/src/normalize/mod.rs

```diff
@@ -2764,7 +2882,12 @@ mod tests {
         }))
         .unwrap();
 
-        normalize_measurements(&mut event, Some(&config), None);
+        let config = DynamicMeasurementsConfig {
+            project: Some(&config),
```

Comment:
The config passed to the project is the project-specific measurement config, could we rename that variable to make it easier to understand?